Dockerfile

```diff
@@ -13,26 +14,36 @@ RUN wget https://nixos.org/releases/nix/nix-${NIX_VERSION}/nix-${NIX_VERSION}-$(
   && addgroup -g 30000 -S nixbld \
   && for i in $(seq 1 30); do adduser -S -D -h /var/empty -g "Nix build user $i" -u $((30000 + i)) -G nixbld nixbld$i ; done \
   && mkdir -m 0755 /etc/nix \
-  && echo 'sandbox = false' > /etc/nix/nix.conf \
-  && mkdir -m 0755 /nix && USER=root sh nix-${NIX_VERSION}-$(uname -m)-linux/install \
+  && printf 'filter-syscalls = false\nsandbox = false\n' > /etc/nix/nix.conf \
```

Comment:
Why do we need filter calls now?

Reply:
I couldn't remember the exact failure so tried to reproduce it again. When running docker build under qemu soft emulation nix just fails to install. I've updated the commit message with the info and a snipped of the failure here: [`609eab8` (#43)](https://github.com/NixOS/docker/pull/43/commits/609eab80faf4681e574323de35d1740eff52ba47)

Reply:
This was also an issue for me when trying to build an arm64 image with Github actions. If I built it on M1 MacBook the build finished fine and I ended up with an arm64 image that i could use.